release_notes/ocp-4-10-release-notes.adoc

```diff
@@ -530,6 +530,12 @@ You can now set the maximum length of the syslog message in the Ingress Controll
 
 For more information, see xref:../networking/ingress-operator.adoc#nw-ingress-controller-configuration-parameters_configuring-ingress[Ingress Controller configuration parameters].
 
+[id="ocp-4.10-ne-set-coredns-forwarding-policy"]
+==== Set CoreDNS forwarding policy
+You can now set the CoreDNS forwarding policy through the DNS operator. The default value is `Random`, you can also set the values to `RoundRobin`, and `Sequential`. 
```

Comment:
I don't think you need a comma between `RoundRobin` and `Sequential`.

Reply:
removing the comma